src/Components/App.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import Areas from '../Components/Areas';
+import  getHomeRepairs  from '../apiCalls';
+import NavBar from './NavBar';
+import Videos from '../Videos';
+import Projects from './Projects';
+import { Route, Switch } from 'react-router-dom'
+import '../App.css';
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      areas: ['kitchen', 'bathroom', 'bedroom', 'miscellaneous'],
+      homeRepairs: []
+    }
+  }
+
+  componentDidMount() {
+    getHomeRepairs()
+    .then(data => this.setState({homeRepairs: data}))
+  }
+
+  returnProjects = (projects, category) => {
+     if(projects.length > 0) {
+      let filteredProjects = projects.filter(project => {
+          let lowerCaseArea = project.areaOfHome.toLowerCase()
+          let lowerCaseCategory = category.toLowerCase()
+            if(lowerCaseCategory === lowerCaseArea) {
+               return project;
+            }
```

Comment:
I'm pretty sure you don't need this last if block in your filter. I believe that you could write this function as follows and it would give you the same results -- minus a couple lines of code -- 

`let filteredProjects = projects.filter(project => {
     let lowerCaseArea = project.areaOfHome.toLowerCase()
     let lowerCaseCategory = category.toLowerCase()
     return lowerCaseCategory === lowerCaseArea
   })`

Maybe try it out if you have the time?